CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- `ReadCall` gets new argument `value: u128` representing native tokens transferred. 
```

Comment:
TODO: needs updating. Now `ExecCall` includes the `<T>` of the result as well. 

@obrok @pmikolajczyk41 , we need to meet and discuss how we want the API to evolve here - do we want to maintain the common types or split into independent crates.